### PR TITLE
Get node status

### LINKED ISSUE #870: Create command for users to check if node is bootstrapped

### PR DESCRIPTION
**Pull request by sukantoraymond**
gets status if node is bootstrapped to primary network or subnet
closes #870

### CODE REVIEW

File: cmd/nodecmd/node.go [hunk 1]
```diff
@@ -32,9 +32,11 @@ rest of the commands to maintain your node and make your node a Subnet Validator
 	app = injectedApp
 	// node create
 	cmd.AddCommand(newCreateCmd())
-	// node join subnet
+	// node join cluster
```
Comment (felipemadero) [inline cmd/nodecmd/node.go:35]: join also has a subnet flag param

File: cmd/nodecmd/join.go [hunk 2]
```diff
@@ -73,6 +83,12 @@ func parseBootstrappedOutput(filePath string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
+	if printOutput {
+		err = printJSONOutput(byteValue)
```
Comment (felipemadero) [inline cmd/nodecmd/join.go:87]: if err

File: cmd/nodecmd/join.go [hunk 3]
```diff
@@ -61,7 +62,16 @@ avalanche node status`,
...
+func printJSONOutput(byteValue []byte) error {
+	var prettyJSON bytes.Buffer
+	if err := json.Indent(&prettyJSON, byteValue, "", "   "); err != nil {
+		return err
+	}
+	ux.Logger.PrintToUser(prettyJSON.String())
+	return nil
+}
+
+func parseBootstrappedOutput(filePath string, printOutput bool) (bool, error) {
```
Comment (felipemadero) [inline cmd/nodecmd/join.go:74]: is this printOutput for debugging no? maybe we can use log level? cc @dasconnor

Comment (felipemadero) [reply cmd/nodecmd/join.go:74]: instead of a boolean

File: cmd/nodecmd/join.go [hunk 4]
```diff
@@ -95,6 +111,12 @@ func parseSubnetSyncOutput(filePath string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	if printOutput {
```
Comment (felipemadero) [inline cmd/nodecmd/join.go:114]: same here, maybe we can use log level for this

File: cmd/nodecmd/status.go [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+import (
+	"github.com/ava-labs/avalanche-cli/cmd/subnetcmd"
+	"github.com/ava-labs/avalanche-cli/pkg/models"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/spf13/cobra"
+)
+
+func newStatusCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "status [subnetName]",
```
Comment (felipemadero) [inline cmd/nodecmd/status.go:14]: cluster

File: cmd/nodecmd/status.go [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+		RunE:         statusSubnet,
+	}
+	cmd.Flags().StringVar(&subnetName, "subnet", "", "specify the subnet the node is syncing with")
+
+	return cmd
+}
+
+func statusSubnet(_ *cobra.Command, args []string) error {
+	clusterName := args[0]
+	err := setupAnsible()
```
Comment (felipemadero) [inline cmd/nodecmd/status.go:31]: if err

File: cmd/nodecmd/status.go [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+	if err != nil {
+		return err
+	}
+	if subnetName != "" {
+		_, err = subnetcmd.ValidateSubnetNameAndGetChains([]string{subnetName})
```
Comment (felipemadero) [inline cmd/nodecmd/status.go:36]: if err

File: cmd/nodecmd/status.go [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+		if err != nil {
+			return err
+		}
+		sc, err := app.LoadSidecar(subnetName)
+		if err != nil {
+			return err
+		}
+		blockchainID := sc.Networks[models.Fuji.String()].BlockchainID
```
Comment (felipemadero) [inline cmd/nodecmd/status.go:44]: can we have a panic here?

File: cmd/nodecmd/status.go [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+		if blockchainID == ids.Empty {
+			return ErrNoBlockchainID
+		}
+		_, err = getNodeSubnetSyncStatus(blockchainID.String(), clusterName, true)
```
Comment (felipemadero) [inline cmd/nodecmd/status.go:48]: do we want to return err of print a ux user message? I think the second option cc @dasconnor

File: cmd/nodecmd/join.go (hunk 3 above)
Comment (dasconnor) [reply cmd/nodecmd/join.go:86]: Yeah, I think this pattern is good

**Review by dasconnor — APPROVED** [review]
LGTM

**Pull request merged**